docs/fuel-101/relationship-to-ethereum.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Relationship to Ethereum
+category: Fuel 101
+parent:
+  label: Fuel 101
+  link: /fuel-101
+---
+
+# Relationship to Ethereum
+
+Fuel is an operating system purpose built for Ethereum Rollups. It allows for rollups to solve for parallelization, state minimized execution and interoperability in Ethereum. Here is how Fuel fits into the Ethereum ecosystem and improves the scalability of the network:
+
+- **Fuel brings long needed improvements to the Ethereum ecosystem.** Due to constraints with backward compatability, the Ethereum virtual machine (EVM) has been limited in its ability to scale. The Fuel virtual machine (FuelVM) brings those improvements to Ethereum.
+- **Fuel reduces Ethereum state bloat.** Fuel utilizes the UTXO model for strict state access lists, which allows for a more efficient usage of state storage on chain.
```

Comment:
```suggestion
- **Fuel reduces Ethereum state bloat.** Fuel utilizes the UTXO model for [strict state access lists](https://github.com/ethereum/EIPs/issues/648), which allows for a more efficient usage of state storage on chain.
```

Reply:
A quick link to the Ethereum EIP for people who don't know what strict access lists are